modes/telega/evil-collection-telega.el

```diff
@@ -160,6 +160,7 @@
     "^" nil
     (kbd "DEL") nil
     "*" nil
+    (kbd "SPC") nil
```

Comment:
`SPC` isn't bound in the past. `scroll-up-command` can be called by `C-f` in evil and some people use `SPC` as there leader key, so I unbind it.

If it is not accepted, I can remove it.

Reply:
Could just leave it instead or removing it. Users that use SPC for leader key should probably have some high powered override map anyways.

Reply:
By "leave it", I mean remove this line.

Reply:
> `SPC` wasn't bound in the past. `scroll-up-command` can be called by `C-f` in evil and some people use `SPC` as there leader key, so I unbind it here.

It will break for the users not using evil leader. Don't do that.